pkg/apiserver/model/application.go

```diff
@@ -213,6 +213,8 @@ type ApplicationRevision struct {
 	EnvName string `json:"envName"`
 	// CodeInfo is the code info of this application revision
 	CodeInfo *CodeInfo `json:"codeInfo,omitempty"`
+	// ImageInfo is the image info of this application revision
+	ImageInfo *ImageInfo `json:"imageInfo,omitempty"`
```

Comment:
Is it necessary?  There seems to be no more information about the image than the fullName.

Reply:
Image Digest, Create Time, Repository Type(public or private), region

Reply:
Can harbor provide these data? It should not be designed to be unique to ACR.

The other thing to consider is whether the user needs this data.

Reply:
> Can harbor provide these data?

Yes, except for `region`